### PR TITLE
LVC-232 (Refactor TCPClient class)

### PR DESCRIPTION
**Pull request by Cv287**
## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- []  PR meets all conventions

### CODE REVIEW

File: RealTimeFaceMaskDetector/TCPClientUI/TCPClient.h [hunk 1]
```diff
@@ -13,51 +13,39 @@
...
 #include "ui_FaceRecognitionUI.h"
 #include "MaskRecognizer.h"
 
 #pragma comment (lib, "ws2_32.lib")
 
+/* Forward declaration. */
 class FaceRecognitionUI;
 
+/* These are global variables.
+   g_ip stores server ip & g_port stores server port.*/
```
Comment (marko-yaciv) [inline RealTimeFaceMaskDetector/TCPClientUI/TCPClient.h]: You can add, where were they defined

File: RealTimeFaceMaskDetector/TCPClientUI/TCPClientUI.cpp [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #include "TCPClientUI.h"
 #include "TCPClient.h"
+#include "../EncryptDecryptECBMode/EncryptDecryptAES_ECBMode.h"
```
Comment (marko-yaciv) [inline RealTimeFaceMaskDetector/TCPClientUI/TCPClientUI.cpp:4]: Here add a description, or at least definition location too

**Review by ChyzhTolik — APPROVED** [review]

File: RealTimeFaceMaskDetector/TCPClientUI/TCPClient.h (hunk 1 above)
Comment (Cv287) [reply RealTimeFaceMaskDetector/TCPClientUI/TCPClient.h]: Thanks for the advice. Done.

File: RealTimeFaceMaskDetector/TCPClientUI/TCPClientUI.cpp (hunk 2 above)
Comment (Cv287) [reply RealTimeFaceMaskDetector/TCPClientUI/TCPClientUI.cpp:4]: Added description for global variables

**Comment by Cv287** [comment]
These changes will be merged to branch face_recognition and then to development.

**Pull request closed without merging**